webhook.rst

```diff
@@ -106,7 +175,195 @@ For Mailgun, you will get a secret for the webhook. Store this secret as
+                $event = new LowPriorityStockRefillEvent($name, $payload['id]', $payload])
+            }
+
+            ....
+
+            return $event;
+        }
+    }
+
+From this, we can see that the RemoteEvent component is highly beneficial for handling webhooks.
+It enables you to convert the incoming webhook data into validated objects that can be efficiently manipulated and utilized according to your requirements.
+
+Remote Event Consumer: Handling and Manipulating The Received Data
+------------------------------------------------------------------
+
+It is important to note that when the incoming webhook is processed by the :class:`Symfony\\Component\\Webhook\\Controller\\WebhookController`, you have the option to handle the consumption of remote events asynchronously.
+Indeed, this can be configured using a bus, with the default setting pointing to the Messenger component's default bus.
+For more details, refer to the :doc:`Symfony Messenger </components/messenger>` documentation
+
+
+Whether the remote event is processed synchronously or asynchronously, you'll need a consumer that implements the :class:`Symfony\\Component\\RemoteEvent\\Consumer\\ConsumerInterface`.
+If you used the command to set this up, it was created automatically
+
+.. code-block:: terminal
+
+    $ php bin/console make:webhook
+
+Otherwise, you'll need to manually add it with the ``AsRemoteEventConsumer`` attribute which will allow you to designate this class as a consumer implementing :class:`Symfony\\Component\\RemoteEvent\\Consumer\\ConsumerInterface`,
+making it recognizable to the RemoteEvent component so it can pass the converted object to it.
+Additionally, the name passed to your attribute is critical; it must match the configuration entry under routing that you specified in the ``webhook.yaml`` file, which in your case is ``my_first_parser``.
+
+In the :method:`Symfony\\Component\\RemoteEvent\\Consumer\\ConsumerInterface::consume` method,
+you can access your object containing the event data that triggered the webhook, allowing you to respond appropriately.
+
+For example, you can use Mercure to broadcast updates to clients of the hub, among other actions ...::
+
+    // src/Webhook/ExampleRequestParser.php
+    #[AsRemoteEventConsumer('my_first_parser')] # routing name
+    final class ExampleWebhookConsumer implements ConsumerInterface
+    {
+        public function __construct()
+        {
+        }
+
+        public function consume(RemoteEvent $event): void
+        {
+            // Implement your own logic here
+        }
+    }
+
+
+If you are using it alongside other components that already include built-in parsers,
+you will need to configure the settings (as mentioned earlier) and also create your own consumer.
+This is necessary because it involves your own business logic and your specific reactions to the remote event(s) that may be received from the built-in parsers.
+
+Usage in Combination with the Mailer Component
```

Comment:
I'm not sure having 2 sections about using it in combination with mailer and notifier in the page (one in the section about the parser and one in the section about the handler) makes sense. I think it might be better to organize the content the other way around:

- usage with mailer
- usage with notifier
- usage with custom webhooks (talking about both the request parser and the handler)